onnxruntime/core/platform/windows/env.h

```diff
@@ -0,0 +1,132 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+// Portions Copyright (c) Microsoft Corporation
+
+#include "core/platform/env.h"
+#include "core/platform/windows/telemetry.h"
+#include <Windows.h>
+
+namespace onnxruntime {
+
+/*
+Logical processor information:
+1. its belonging group;
+2. its id within that belonging group;
+*/
+struct ProcessorInfo {
+  int group_id;
+  int local_processor_id;
+};
+
+using GlobalProcessorInfoMap = std::unordered_map<int, ProcessorInfo>;
```

Comment:
>GlobalProcessorInfoMap [](http://example.com/codeflow?start=6&length=22)

Add comment to this - 
"GlobalProcessorInfoMap" is a map between global processor id and pair<groupid, local processor id>, the latter are required to be present during affinity setup.